src/map.cpp

```diff
@@ -417,7 +417,7 @@ bool map::vehact( vehicle &veh )
 
     // "Anti-ideal" vehicle slows down up to 10 times faster than ideal one
     const float k_slowdown = 20.0f / ( 2.0f + 9 * ( veh.k_dynamics() * veh.k_mass() ) );
-    const int slowdown = veh.drag() + (int)ceil( k_slowdown * base_slowdown );
+    const float slowdown = veh.drag() + (int)ceil( k_slowdown * base_slowdown );
```

Comment:
Now that the result is stored as floating point value, the cast to `int` and the call to `ceil` (probably inserted for the conversion) may not be required any more?

Reply:
Probably.